source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp

```diff
@@ -73,19 +73,25 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
 
             double cellSpacingDouble = static_cast<double>(tableConfig.CellSpacing());
 
-            // Set left and top margin for each cell (to avoid double margins). Don't set the margin on topmost
-            // or leftmost cells to avoid creating margin outside the table.
-            winrt::Thickness marginThickness = {cellSpacingDouble, cellSpacingDouble, 0, 0};
-            if (columnNumber == 0)
+            if (cellSpacingDouble != 0)
             {
-                marginThickness.Left = 0;
+                // Set left and top margin for each cell (to avoid double margins). Don't set the margin on topmost
+                // or leftmost cells to avoid creating margin outside the table.
+                winrt::Thickness marginThickness = {cellSpacingDouble, cellSpacingDouble, 0, 0};
+                if (columnNumber == 0)
```

Comment:
nit: spacing isn't consistent; between if statements; try running clang-format. it should fix it for you.

Reply:
I ran clang-format and removed some of the empty lines.